test/clt-tests/core/test-fuzzy-search.rec

```diff
@@ -0,0 +1,569 @@
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 2 AS fuzziness, 'us,ru' AS layouts, 1 AS append, 1 AS prepend, 2 AS expansion_len);" | tail -n +4 | sort
+––– output –––
++-------+
+| joe   |
+| john  |
+| jose  |
+| joy   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 1 AS append);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 1 AS prepend);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 1 AS append, 1 AS prepend);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 'us' AS layouts);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 'ru' AS layouts);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('al', 'name', 0 AS fuzziness);" | tail -n +4 | sort
+––– output –––
++-----------+
+| albert    |
+| alexander |
+| allen     |
+| dale      |
+| dallas    |
+| donald    |
+| gale      |
+| kendall   |
+| marshall  |
+| walter    |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('mi', 'name', 0 AS append, 1 AS prepend);" | tail -n +4 | sort
+––– output –––
++---------+
+| ami     |
+| britni  |
+| brittni |
+| cami    |
+| choi    |
+| darci   |
+| eleni   |
+| jami    |
+| jani    |
+| jeni    |
+| joi     |
+| kaci    |
+| kami    |
+| kimi    |
+| laci    |
+| lani    |
+| luci    |
+| mancini |
+| marci   |
+| martini |
+| melani  |
+| muoi    |
+| nanci   |
+| neomi   |
+| noemi   |
+| nohemi  |
+| ricci   |
+| shani   |
+| staci   |
+| tammi   |
+| toi     |
+| tomi    |
+| traci   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('ch', 'name', 1 AS fuzziness, 3 AS expansion_len);" | tail -n +4 | sort
+––– output –––
++-------+
+| chang |
+| chi   |
+| chung |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('pa', 'name', 'us' AS layouts);" | tail -n +4 | sort
```

Comment:
The test is useless because the layouts are shifting. We should review these tests and conduct proper testing. For example, the layouts test is about guessing the keyboard layout.

We pass 'us,ru' as layouts and test with `зф`, which should translate to -> `pa`.

When the layouts include Russian (ru), it should work.
When Russian is not included, it should not work.

Reply:
Done